stdlib/ballerina-http/src/main/ballerina/http/http_load_balancer.bal

```diff
@@ -307,7 +307,7 @@ function performLoadBalanceAction (LoadBalancer lb, string path, Request outRequ
         match outRequest.getBinaryPayload() {
             // TODO: remove these dummy assignments once empty blocks are supported
             blob => {int x = 0;}
-            mime:EntityError => {int x = 0;}
+            error => {int x = 0;}
```

Comment:
Can you get rid of these dummy assignments as well please? These are left over stuff from the early days of the refactor